src/AppHost.ts

```diff
@@ -182,17 +178,6 @@ export enum Command {
     TARGET_BLANK = "targetblank",
 }
 
-/**
- * Available layouts
- *
- * See also: {@link IAppHost.getDefaultLayout}
- */
-export enum Layout {
```

Comment:
The enum was important because it could be imported from js

Reply:
Enums generally look really bad on DocFX:

![image](https://user-images.githubusercontent.com/19396809/79549380-307ae980-8097-11ea-900e-94207bbdb77c.png)

The primary purpose of this would be to generate documentation for client developers wanting to implement a NativeShell-based client, so they need to have the proper return values in the docs, if these are limited to a certain set of values, imo.

Reply:
Well to me it looks okay, rather make a issue for docfx

Reply:
Also the supported commands are only available in this enum there'd be no docs for them if it got removed